packages/orbit-components/src/Modal/ModalHeader/index.tsx

```diff
@@ -24,7 +24,8 @@ export const ModalHeaderWrapper = ({
       className={cx(
         className,
         "orbit-modal-header-container",
-        "box-border block w-full",
+        "rounded-t-modal box-border block w-full",
+        "lm:[&_~_.orbit-modal-section]:rounded-t-none",
```

Comment:
Be default, a modal-section has rounded top corners. But when the Modal has a header, those corners can't be rounded. This class prevents that.